CHANGELOG.md

```diff
@@ -1,3 +1,89 @@
+# Release 0.16.1
+
+In addition, the security of the Scope probe can be hardened by disabling
```

Comment:
It used to be: _Bugfix release. In addition ..._ I removed the first phrase by error when addressing a PR comment. Will fix